ArtId/src/main/java/TestProjj/ArtId/App.java

```diff
@@ -15,10 +20,88 @@ public static void main( String[] args )
         new Profiler.Builder()
         .profilingGroupName("CodeGuruTestProfileGroup")
         .build().start();
-        System.out.println( "Testing it out" );
-        System.out.println( "Testing it out" );
-        System.out.println( "Testing it out" );
-        System.out.println( "Testing it out" );
-        System.out.println( "Testing it out" );
+        
+        new Profiler.Builder()
+        .profilingGroupName("CodeGuruTestProfileGroup")
+        .build().start();
+        
+        new Profiler.Builder()
+        .profilingGroupName("CodeGuruTestProfileGroup")
+        .build().start();
+        
+        new Profiler.Builder()
+        .profilingGroupName("CodeGuruTestProfileGroup")
+        .build().start();
+        
+        new Profiler.Builder()
+        .profilingGroupName("CodeGuruTestProfileGroup")
+        .build().start();
+        
+        new Profiler.Builder()
+        .profilingGroupName("CodeGuruTestProfileGroup")
+        .build().start();
+        
+        new Profiler.Builder()
+        .profilingGroupName("CodeGuruTestProfileGroup")
+        .build().start();
+        
+        new Profiler.Builder()
+        .profilingGroupName("CodeGuruTestProfileGroup")
+        .build().start();
+        
+        new Profiler.Builder()
+        .profilingGroupName("CodeGuruTestProfileGroup")
+        .build().start();
+        
+        new Profiler.Builder()
+        .profilingGroupName("CodeGuruTestProfileGroup")
+        .build().start();
+        
+        System.out.println( "Done man done !!!!" );
+        
+        
+        String fileName = "intData.dat" ;   long sum = 0;
+
+        try
+        {      
+          DataInputStream instr = 
+            new DataInputStream(
+              new BufferedInputStream(
+                new FileInputStream( fileName  ) ) );
+
+          sum += instr.readLong();
+          sum += instr.readLong();
+          System.out.println( "The sum is: " + sum );
+          instr.close();
+        }
+        catch ( IOException iox )
+        {
+          System.out.println("Problem reading " + fileName );
+        }
+        
+        
+        System.out.println( "Hello World!" );
+        System.out.println( "Hello World!" );
+        System.out.println( "Hello World!" );
+        System.out.println( "Hello World!" );
+        
+        String fileName1 = "intData.dat" ;   long sum1 = 0;
+
+        try
+        {      
+          DataInputStream instr = 
+            new DataInputStream(
+              new BufferedInputStream(
+                new FileInputStream( fileName1  ) ) );
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

**Problem**
This line of code contains a resource that might not be closed properly. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the resource created by the following constructor call: *FileInputStream*. The resource is referenced in statements at the following lines: 93, 94, 97, 98. The resource closure statement is at line: 100. There are other execution paths that do not contain closure statements, e.g., when exception is thrown by DataInputStream.readLong. Either a) close the object returned by *FileInputStream()* in a try-finally block or b) close the resource by declaring the object returned by *FileInputStream()* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).